E-Commerce/readme.md

```diff
@@ -46,13 +46,47 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+
+```sql
+SELECT * FROM products WHERE lower(product_name) LIKE '%socks%';
+```
+
 - [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+
+```sql
+SELECT p.id, p.product_name, pa.unit_price, s.id FROM products p INNER JOIN product_availability pa ON p.id = pa.prod_id INNER JOIN suppliers s ON s.id = pa.supp_id WHERE pa.unit_price > 100;
+```
```

Comment:
CORRECT - although consider spreading your query over multiple lines, for readability